grails-app/i18n/messages.properties

```diff
@@ -34,6 +34,7 @@ search.header.kingdom.Fungi = Fungi
 search.header.kingdom.Plantae = Plants
 search.header.kingdom.Protista = Protists
 search.download.link.title = Download a list of taxa for your search
+search.errorMsg = Search failed. Please check your inputs and try again
```

Comment:
Period's missing.

Reply:
look around. do you see any ending periods anywhere else?

Reply:
This is different becasue you have two sentences here. If one's ending with a period, so should the other.

You don't have to use periods if the element consists of one label or statement.

Reply:
the last sentence doesn't need to end with a perion